spec/supabase_js_v2_legacy.yml

```diff
@@ -509,23 +509,51 @@ pages:
         isSpotlight: true
         js: |
           ```js
-          const { data, error } = await supabase.auth.admin.generateLink(
-            'email@example.com'
-            'signup',
-            {
+          const { data, error } = await supabase.auth.admin.generateLink({
+            type: 'signup',
+            email: 'email@example.com'
+            options: {
               'password': 'secret'
             }
-          )
+          })
           ```
       - name: Generate an invite link.
```

Comment:
```suggestion
      - name: Generate an invite link
```

Reply:
@dannykng - Just removed all of the periods and formatted.